src/webgl/loading.js

```diff
@@ -26,6 +26,11 @@ import './p5.Geometry';
  * `loadModel('assets/model.obj')`. URLs such as
  * `'https://example.com/model.obj'` may be blocked due to browser security.
  *
+ * Note: When loading a `.obj` file that references materials stored in
+ * `.mtl` files, p5.js will attempt to load and apply those materials.
+ * To ensure that the `.obj` file reads the `.mtl` file correctly include the 
+ * `.mtl` file alongside it. Currently, other formats like `.lib` for materials are not supported.
```

Comment:
Sorry for the delay! One last thing I think, I'd rather not mention `.lib` here because I can't actually find resources about it in the context of 3D object `.obj` files online (only in the context of c++ compilation .obj and .lib files, which are unrelated), so I think its inclusion might just be more confusing to the average user than it's worth. The rest looks good though! So I'm just going to remove that last sentence and merge.

```suggestion
 * `.mtl` file alongside it.
```